docs/general-information/project-template.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Project Template
+parent: General Information
+---
+
+# Project Template
+
+## Overview
+
+The project template is a base project with all the basic functionality built in for developing
+Python or R projects. This helps save time and effort at the beginning of a project and keeps
+project development workflows and standards consistent across the lab. The base project uses
+[Docker](https://ndclab.github.io/wiki/docs/technical-docs/docker-usage.html) as a development
+platform and as a deployment platform.
+
+By default the project contains a project directory, markdown templates for issues and pull
+requests, shell scripts to automate some basic actions, and yaml configurations for [GitHub
+Actions].
+
+## Usage
+
+To use the template first clone the project template
+[repository](https://github.com/NDCLab/project-template/tree/main).
+
+Once the repository has been cloned, open a terminal and `cd` into the directory.
+
+Run the `init.sh` [Bash](https://en.wikipedia.org/wiki/Bash_%28Unix_shell%29) script:
+
+```bash
+# "name" is the name of the project
+# "name" should be all lower case, and contain no spaces
+./init.sh name
+```
+
+To create the [Docker container](https://www.docker.com/resources/what-container), run
+`docker-compose up -d --build`. This will run the [Docker Compose](https://docs.docker.com/compose/)
+configuration file and automatically build the Docker container and run it.
+
+> Note: The docker container will automatically mount the `data` and `project_name` directory. This
```

Comment:
Say briefly what the "data" and "project_name" directories are/are for.

Reply:
...unless already specified in docker doc. Maybe it should be specified there as well if not?